assets/stylesheets/sections/comments.scss

```diff
@@ -0,0 +1,10 @@
+/** @format */
+@import '../shared/colors';
+@import '../shared/functions/z-index';
+@import '../shared/mixins/breakpoints';
+@import '../shared/mixins/placeholder';
+@import '../shared/typography';
```

Comment:
I've noticed that some of these shared assets are imported via `@import 'shared/utils';` in the original PR. Should we consider doing something similar here? (and add the missing ones to `shared/utils`)

Reply:
They are loaded as `shared/utils` in the original PR, but not in the current files (e.g. https://github.com/Automattic/wp-calypso/blob/master/assets/stylesheets/sections/post-editor.scss#L4-L12).

I'm not exactly sure why (the commit message is just "Editor: Import only required stylesheets for section"), so I guess it's to avoid importing unnecessary assets.